ui/details-panel/FailureSummaryTab.jsx

```diff
@@ -0,0 +1,263 @@
+import PropTypes from 'prop-types';
+import createHistory from 'history/createBrowserHistory';
+
+import treeherder from '../js/treeherder';
+import { getBugUrl } from '../helpers/urlHelper';
+import { getAllUrlParams } from '../helpers/locationHelper';
+import { escapeHTML, highlightCommonTerms } from "../helpers/displayHelper";
+
+const BUG_LIMIT = 20;
+
+class SuggestionsListItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      suggestionShowMore: false,
+      filerInAddress: getAllUrlParams().has('bugfiler'),
+    };
+
+    this.updateFilerInAddress = this.updateFilerInAddress.bind(this);
+    this.clickShowMore = this.clickShowMore.bind(this);
+  }
+
+  componentWillMount() {
+    this.unlistenHistory = this.props.history.listen(this.updateFilerInAddress);
+  }
+
+  componentWillUnmount() {
+    this.unlistenHistory();
+  }
+
+  updateFilerInAddress() {
+    this.setState({ filerInAddress: getAllUrlParams().has('bugfiler') });
+  }
+
+  clickShowMore() {
+    this.setState({ suggestionShowMore: !this.state.suggestionShowMore });
+  }
+
+  render() {
+    const {
+      user, suggestion, selectedJob, $timeout, pinboardService, fileBug, index
+    } = this.props;
+    const { filerInAddress, suggestionShowMore } = this.state;
+
+    return (
+      <li>
+        <div className="job-tabs-content">
+          {(filerInAddress || user.is_staff) &&
```

Comment:
It might be worth seeing if we can drop the requirement of only showing the bug filer to is_staff? (Thereby making the `filerInAddress` unnecessary). Or if not, then perhaps we can get rid of `filerInAddress` anyway? (It's not clear anyone would even know about this feature who wasn't already a sheriff)

Reply:
If I'm following you, you're suggesting just making it always visible for everyone?  That's fine by me.  :)

Reply:
yeah (the argument against that at the time was that people would file bugs for things that weren't really intermittents, but it seems like something we should wait until proven before worrying about it. And even if it becomes an issue, we could handle in other ways)